controllers/article.controllers.js

```diff
@@ -0,0 +1,27 @@
+const {
+  fetchArticleById,
+  updateArticleById,
+} = require("../models/article.models");
+
+exports.getArticleById = (req, res, next) => {
+  const id = Object.values(req.params);
```

Comment:
You don't need Object.values here (but it works, so if you want to leave it in, I'm not mad at it). You could just go for const {article_id} = req.params to destructure it straight from the params object.

Reply:
The above would also stop you having to use the 0 index on line 8.